### DISCUSSION TITLE
Eglot / dumb-jump

### DISCUSSION THREAD

**Discussion by konrad1977** [joaotavora/eglot#1024] [Q&A] [answered]
Hi!
Eglot takes over my go-to-definition (g d in Evil) when enabled. Thats probably fine in most language servers but the swift-lsp server is so bad and cant find the references very good (not at all for most time).

`"No defintion found for: LSP identifier at point"`

Can eglot be configured to not do this some how? Or return nil, so xref can use next handler/backend/function for this?

**Comment by jgarte** [comment]
I think you're referring to evil-collection and `xref-find-definitions` being bound to `gd` in `eglot-mode-map` when eglot is enabled: https://github.com/emacs-evil/evil-collection/blob/master/modes/eglot/evil-collection-eglot.el#L38

Is that correct?

**Reply by konrad1977** [reply]
Yes, I still want to use dumb-jump for 'gd', because the Swift LSP support is really poor and doesnt work.

**Comment by konrad1977** [comment, marked as answer]
This solved my issue: https://github.com/joaotavora/eglot/issues/420#issuecomment-1257247512
